adafruit_pn532/i2c.py

```diff
@@ -49,6 +49,42 @@ def __init__(
         """Create an instance of the PN532 class using I2C. Note that PN532
         uses clock stretching. Optional IRQ pin (not used),
         resetp pin and debugging output.
+
+        :param busio.I2C i2c: The I2C bus the PN532 is connected to.
+        :param int address: The I2C device address. Defaults to :const:`0x24`
+        :param DigitalInOut reset: board pin the PN532 reset is connected to
+        :param DigitalInOut req: board pin the PN532 P32 is connected to
+        :param bool debug: if true print additional debug statements
+
```

Comment:
If any parameter defaults to a specific value. we indicate this with
.Defaults to xxxx.
for example for
:param bool debuf: if `True` print additional debug statemens. Defaults to `False`